src/utils/AccurateMassSearch.cpp

```diff
@@ -99,9 +102,10 @@ class TOPPAccurateMassSearch :
     setValidFormats_("in", {"featureXML", "consensusXML"});
     registerOutputFile_("out", "<file>", "", "mzTab file");
     setValidFormats_("out", ListUtils::create<String>("mzTab"));
```

Comment:
@timosachsenberg: FileHandling not only on reading but more on writing MzTab, since only "mzTab" is registered in the FileHandler and not "mztabm" - due to the same suffix. So handling if MzTab or MzTab-M should be written in the end is currently handled by the tool e.g. via a flag (here -mztabm).

Reply:
why is the flag needed?

Reply:
you want to support both?

Reply:
Yes, at least in a transition period - I think we should support both formats since a lot of older post-processing workflows are dependent on the MzTab format.

Reply:
maybe I would prefer having a string option -mzTab_version with valid strings "mzTab-1.0.0" and "mzTab-M-v1.0.0" ? what do you think?

Reply:
Yes, that would be fine as well :). I was only thinking if that could somehow be handled at the FileHandler level directly without needing the implementation for every tool (it applies to).